uber/templates/shifts_admin/staffers.html

```diff
@@ -9,7 +9,7 @@
     <div class="col-md-3">{{ counts.regular_signups }} / {{ counts.regular_total }} Regular Hours Taken</div>
     <div class="col-md-3">{{ counts.restricted_signups }} / {{ counts.restricted_total }} Restricted Hours Taken</div>
     <div class="col-md-3">{{ counts.all_signups }} / {{ counts.all_total }} Total Hours Taken</div>
-</div>
+</div>{{ c.DEFAULT_DEPARTMENT_ID }}
```

Comment:
This looks like something you put here for debugging purposes and neglected to remove.

Reply:
Heck!